server/src/js/content-helper.js

```diff
@@ -8,4 +8,16 @@ window.addEventListener(
     window.location.origin)
 );
 
+window.addEventListener(
+  "message",
+  (m) => {
+    let node = document.getElementById(m.data.show),
```

Comment:
We should be doing some assertion here, I think that `m.origin == location.origin` (i.e., the message comes from our site)